x-pack/plugins/enterprise_search/server/routes/enterprise_search/connectors.ts

```diff
@@ -73,17 +73,8 @@ export function registerConnectorRoutes({ router, log }: RouteDependencies) {
     },
     elasticsearchErrorHandler(log, async (context, request, response) => {
       const { client } = (await context.core).elasticsearch;
-      try {
-        await updateConnectorConfiguration(client, request.params.connectorId, request.body);
```

Comment:
I think I missed a PR somewhere. Can I assume these are handled properly on frontend now instead ?

Reply:
The `elasticsearchErrorHandler` that wraps these functions now handles errors so that we have a consistent experience. That was this PR: https://github.com/elastic/kibana/pull/138516